src/cpu/minor/execute.cc

```diff
@@ -876,6 +876,15 @@ Execute::doInstCommitAccounting(MinorDynInstPtr inst)
     cpu.commitStats[inst->id.threadId]->numOps++;
     cpu.commitStats[inst->id.threadId]
         ->committedInstType[inst->staticInst->opClass()]++;
+    if (inst->isInst()) {
```

Comment:
Why do you need to check `isInst()`? How could that be false?

Reply:
I initially put that in the issue logic, since doing `inst->staticInst->isInteger()` would segfault without it. Otherwise, I'll take it out.

Only question I have about adding additional stats from `base.cc` of the base CPU is, does Minor have an equivalent execcontext? I noticed that within `BaseExecContext`, there are specific stats that are updated (such as `numCallsReturns`, which I assume I'd have to do something similar and create an exec context to access the thread's info, although there could be a better approach?

Reply:
@poal023, you still need to address Jason's question. Please do so, so we can move this PR closer to being merged. Thank you.

Reply:
My latest comment removes this if statement.